File: module_wanandroid/src/main/java/com/yyxnb/module_wanandroid/ui/WanActivity.java
package com.yyxnb.module_wanandroid.ui;

import android.support.v4.app.Fragment;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.yyxnb.common_base.core.ContainerActivity;

import static com.yyxnb.common_res.arouter.ARouterConstant.WAN_MAIN;

@Route(path = WAN_MAIN)
public class WanActivity extends ContainerActivity {

    @Override
    public Fragment initBaseFragment() {
        return new WanMainFragment();
    }

}

File: module_wanandroid/src/main/java/com/yyxnb/module_wanandroid/ui/WanActivity.kt
package com.yyxnb.module_wanandroid.ui

import androidx.fragment.app.Fragment
import com.alibaba.android.arouter.facade.annotation.Route
import com.yyxnb.common_base.arouter.ARouterConstant.WAN_MAIN
import com.yyxnb.common_base.base.ContainerActivity

@Route(path = WAN_MAIN)
class WanActivity : ContainerActivity() {

    override fun initBaseFragment(): Fragment? {
        return WanMainFragment()
    }
}